producer_sarama.go

```diff
@@ -55,6 +55,7 @@ func NewSaramaProducer(conf *ProducerConfig) Producer {
 	config.RequiredAcks = sarama.RequiredAcks(conf.Acks)
 	config.RetryBackoff = conf.RetryBackoff
 	config.Timeout = conf.Timeout
+	config.AckSuccesses = conf.AckSuccesses
```

Comment:
how about errors? we need to give the option to the operator what todo on error and how that should commit the offset or not